pkg/controller/multiarchbuildconfig/multiarchbuildconfig.go

```diff
@@ -61,32 +64,25 @@ func AddToManager(mgr manager.Manager, architectures []string, dockerCfgPath str
 		UpdateFunc:  func(event.UpdateEvent) bool { return true },
 		GenericFunc: func(event.GenericEvent) bool { return false },
 	}
-	if err := c.Watch(source.Kind(mgr.GetCache(), &v1.MultiArchBuildConfig{}), mabcHandler(), predicateFuncs); err != nil {
+
+	if err := c.Watch(source.Kind(mgr.GetCache(), &v1.MultiArchBuildConfig{}),
+		&handler.EnqueueRequestForObject{}, predicateFuncs); err != nil {
 		return fmt.Errorf("failed to create watch for MultiArchBuildConfig: %w", err)
 	}
 
 	return nil
 }
 
-func mabcHandler() handler.EventHandler {
```

Comment:
Why this is deleted?

Reply:
It's redundant since `source.Kind(mgr.GetCache(), &v1.MultiArchBuildConfig{})` already guarantees to filter objects of type `v1.MultiArchBuildConfig{}` and then `&handler.EnqueueRequestForObject{}` creates a new `reconcile.Request` with the proper object name and namespace. [Here](https://github.com/kubernetes-sigs/controller-runtime/blob/658c55227830b2d895b12fc1c86cbc731e36d291/examples/builtins/main.go#L62) an example from the `controller-runtime` repo.